packages/main/src/plugin/util/exec.ts

```diff
@@ -64,6 +64,15 @@ export class Exec {
 
           // convert process.env to { [key: string]: string; }'
           const sudoEnv = env as { [key: string]: string };
+          /*
+           * sudo prompt verify keys and does not support keys with special characters
```

Comment:
this is such an odd bug, is there any issue open upstream / we can link here for sudo prompt?

Reply:
> this is such an odd bug, is there any issue open upstream / we can link here for sudo prompt?

Looks like a feature: https://github.com/jorangreef/sudo-prompt/blob/c3cc31a51bc50fe21fadcbf76a88609c0c77026f/index.js#L96

And repo is not active enough but will open an issue